main/ejml-core/src/org/ejml/ops/DMonoid.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2009-2020, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of Efficient Java Matrix Library (EJML).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ejml.ops;
+
+public class DMonoid {
+    // possible re-interpreting 0
```

Comment:
This is somewhat difficult to understand. I think what it means is that it is possible to define a different identity element (zero value) instead of `0.0d`. Right?

Reply:
Right. Maybe I could just mention `neutral element` as a synonym? At least in my linear algebra class, we more often used `neutral element` than `identity`.

Reply:
Yes, IMHO neutral element is definitely a good term to use: https://en.wikipedia.org/wiki/Identity_element
The GraphBLAS docs use "identity element" exclusively, so maybe it's good to mention both.

Reply:
When the literature can't make up it's mind I tend to at least add a JavaDoc comment with all the commonly used terms so that someone searching the code can find what they are looking for.